_posts/2016-08-26-babili.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title:  "Releasing Babili (beta)"
+author: Henry Zhu
+date:   2016-08-26 09:30:00
+categories: announcements
+share_text: "Releasing Babili (beta)"
+---
+
+# Babili (babel-minify)
+
+We released [babili](https://github.com/babel/babili) as beta (0.0.1) yesterday under an MIT license!
+
+There are a lot of (valid) questions about why a new minifier is a good idea, so this post should help with that. We also have a [#minify](https://babeljs.slack.com/messages/minify/) slack room!
+
+TL;DR: Babili can output to ES2015+ code so you don't need to transpile and then minify if it's not necessary. Babili is also modular/flexible (it's a babel preset and thus supports user plugins) and can be used as a preset or cli tool. Babili should also be able to do ES2015+ specific optimizations.
+
+## Prounciation
+
+```bash
+# sounds like "bah billy"
+say babili
+```
+
+
+> If you can't remember the name, babel-minify works too.
+
+## Why minify?
+
+[Minification](https://en.wikipedia.org/wiki/Minification_(programming)) removes unncessary characeters without changing its functionality. The basics of minification remove things like comments, whitespace, newlines, and extra parentheses. You can also simplify code, transform code to smaller equivalents, remove unused code, etc.
```

Comment:
Consider this modified paragraph instead:

At a basic level, minification removes unnecessary characters from a program without changing its functionality --  things like comments, whitespace, newlines, and extra parentheses. Advanced minification work by transforming programs into smaller equivalents and by removing redundant or unreachable code.